md/tomcat-bundle.md

```diff
@@ -6,10 +6,10 @@ The Tomcat bundle is a .zip archive that contains the Apache Tomcat JEE applicat
 The Tomcat bundle is a regular .zip archive based on Tomcat zip distribution.
 
 
-::: warning  
+::: warning
 There is a known issue between Bitronix (the Transaction Manager shipped by Bonitasoft in the Tomcat bundle and in the Tomcat directories of the Deploy bundle) and the Microsoft SQL Server driver
```

Comment:
Remove " and in the Tomcat directories of the Deploy bundle": this page only cover Tomcat bundle.
Microsoft SQL Server **JDBC** driver